lib/routes/cjlu/yjsy/index.ts

```diff
@@ -0,0 +1,107 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+const host = 'https://yjsy.cjlu.edu.cn/';
+
+const titleMap = new Map([
+    ['yjstz', '中量大研究生院 —— 研究生通知'],
+    ['jstz', '中量大研究生院 —— 教师通知'],
+]);
+
+export const route: Route = {
+    path: '/yjsy/:cate',
+    categories: ['university'],
+    example: '/cjlu/yjsy/yjstz',
+    parameters: {
+        cate: '订阅的类型，支持 yjstz（研究生通知）和 jstz（教师通知）',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            title: '研究生通知',
+            source: ['yjsy.cjlu.edu.cn/index/yjstz/:suffix', 'yjsy.cjlu.edu.cn/index/yjstz.htm'],
+            target: '/yjsy/yjstz',
+        },
+        {
+            title: '教师通知',
+            source: ['yjsy.cjlu.edu.cn/index/jstz/:suffix', 'yjsy.cjlu.edu.cn/index/jstz.htm'],
+            target: '/yjsy/jstz',
+        },
+    ],
```

Comment:
Since there're only two choices of categories, I list them all in the radar rules.